packages/foreman/foreman/foreman.spec

```diff
@@ -158,13 +159,15 @@ BuildRequires: %{?scl_prefix_ruby}rubygem(rake) >= 0.8.3
 BuildRequires: %{?scl_prefix_ruby}rubygem(rdoc)
 BuildRequires: %{?scl_prefix}rubygem(bundler_ext)
 # start specfile sqlite BuildRequires
```

Comment:
Perhaps we need a fix to start dropping these?

Reply:
Agreed. But that should happen in core first?

Reply:
IIRC we decided to first make sure everything works properly with nulldb, but looks like it does, so definitely.

Reply:
I was indeed saying that it should happen in core and then packaging will follow.

Reply:
opened https://github.com/theforeman/foreman/pull/7617 for core